ccapi/handlers/invokeGateway.go

```diff
@@ -0,0 +1,101 @@
+package handlers
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/goledgerdev/ccapi/chaincode"
+	"github.com/goledgerdev/ccapi/common"
+	"github.com/pkg/errors"
+)
+
+func InvokeGatewayDefault(c *gin.Context) {
+	channelName := os.Getenv("CHANNEL")
+	chaincodeName := os.Getenv("CCNAME")
+
+	invokeGateway(c, channelName, chaincodeName)
+}
+
+func InvokeGatewayCustom(c *gin.Context) {
+	channelName := c.Param("channelName")
+	chaincodeName := c.Param("chaincodeName")
+
+	invokeGateway(c, channelName, chaincodeName)
+}
+
+func invokeGateway(c *gin.Context, channelName, chaincodeName string) {
+	// Get request body
+	req := make(map[string]interface{})
+	err := c.BindJSON(&req)
+	if err != nil {
+		common.Abort(c, http.StatusBadRequest, err)
+		return
+	}
+
+	txName := c.Param("txname")
+
+	// Get endorsers names
+	var endorsers []string
+	endorsersQuery := c.Query("@endorsers")
```

Comment:
@mikaellafs What's the `@endorsers`? Can you explain it or provide a reference link for me?

Reply:
You can explicitly select the organizations to endorse a transaction. Check this [Targeting specific endorsement peers](https://hyperledger-fabric.readthedocs.io/en/latest/gateway.html#targeting-specific-endorsement-peers)